ACMILabs/spotlights
# #66 Update npm dependencies

Resolves #66

Update npm dependencies to fix security alerts: https://github.com/ACMILabs/spotlights/security/dependabot

### Acceptance Criteria
- [x] Update npm dependencies by running `npm update`

### Relevant design files
* None

### Testing instructions
1. See js & python tests pass for this branch: https://github.com/ACMILabs/spotlights/actions
2. Run `npm audit` and see that there aren't any vulnerabilities

### DoD
For requester to complete:
